@tc/webhooks docs (#9170)
expo/expo: docs/pages/distribution/building-standalone-apps.md

### Setting up a webhook

> If you would like to, we can also alert you via a webhook once the build has finished. Webhooks need to be configured per-project, so if you want to be alerted about builds for both `@yourUsername/awesomeApp` and `@yourUsername/coolApp`, you need to run `expo webhooks:add --event build --url <webhook-url>` in each directory.

Reviewer: ```suggestion
Expo can also alert you once your build has finished via a webhook. Webhooks need to be configured per-project, so if you want to be alerted about builds for both `@yourUsername/awesomeApp` and `@yourUsername/coolApp`, you need to run `expo webhooks:add --event build --url <webhook-url>` in each directory.
```
